chia/cmds/wallet_funcs.py

```diff
@@ -68,6 +100,35 @@ async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerpri
                 break
 
 
+async def get_csv(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    wallet_id = args["id"]
+    offset = args["offset"]
+    file_path = args["file"]
+    date_fmt = args["date_fmt"]
+
+    txs: List[TransactionRecord] = await wallet_client.get_csv(wallet_id)
+    config = load_config(DEFAULT_ROOT_PATH, "config.yaml", SERVICE_NAME)
+    name = config["network_overrides"]["config"][config["selected_network"]]["address_prefix"]
+    if len(txs) == 0:
+        print("There are no transactions to this address")
+    
+    file_header = "wallet_id,dateTime,txType,status,heightConfirmed,txName,amount,fee,to_address\n"
+
+    if file_path:
+        f = open(file_path, "w")
+        f.write(file_header);
+        for i in range(offset, len(txs)):
+            f.write( get_csv_line(txs[i], name, date_fmt) )
+            f.write("\n")
+        f.close()
+        print(f"CSV Output to: {file_path}")
+        return
+
+    print(file_header)
+    for i in range(offset, len(txs)):
+        print( get_csv_line(txs[i], name, date_fmt) )
```

Comment:
I'd compute the list of strings upfront.

Reply:
Awesome! Thanks for all your help!  I have implemented your suggestions.  Is there a way I can run that failed linter test locally before committing the latest changes on the PR?  Or is it ok to keep committing changes until all tests pass here?  Thanks again!